desc/equilibrium/equilibrium.py

```diff
@@ -1205,6 +1211,126 @@ def need_src(name):
         )
         return data
 
+    def compute_magnetic_field(
+        self,
+        coords,
+        params=None,
+        basis="rpz",
+        source_grid=None,
+        transforms=None,
+        chunk_size=None,
+    ):
+        """Compute magnetic field at a set of points.
```

Comment:
be more specific: this is the magnetic field generated by the plasma currents.

I think also we could add a flag for external evaluation or not, that will allow the use of the virtual casing principle if desired (if one always knows the evaluation points are far from the plasma surface, like is the case with a lot of diagnostic flux loops)